test/include/seqan3/test/range/container_test_template.hpp

```diff
@@ -5,19 +5,18 @@
 // shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
 // -----------------------------------------------------------------------------------------------------
 
```

Comment:
Why is this now a header?
The templates are usually part of the respective directory of `test`

Reply:
Yeah, here is the problem that it so spread out, that I think that it might be good just move it to the test header.

I thought about this for some time already, and I think that all test templates should be in the test header

This would enable us to use absolute paths `#include <seqan3/test/...>` that will not break when moving test files.

Reply:
Mhm ok, we should also do this for the other templates then.

Reply:
I'll do that as follow-up :)